README.md

```diff
@@ -118,22 +118,41 @@ The analyze mode emits a list of capabilities often seen in malware, categorized
 Requirements:
 
 * [go](https://go.dev/) - the programming language
+* [rust](https://www.rust-lang.org) - yara-x requirement
 * [yara-x](https://virustotal.github.io/yara-x/) - the rule language
 * [pkgconf](http://pkgconf.org/) - required by Go to find C dependencies, included in many UNIX distributions
 
-`yara-x` requires an underlying C API to function. To build and install the API, reference the documentation here: https://virustotal.github.io/yara-x/docs/api/c/c-/#building-the-c-library.
-
-Running `cargo cinstall -p yara-x-capi --release` without `sudo` may encounter permission denied errors.
-
-If this is the case, run the following:
-```sh
-sudo -E env "PATH=$PATH" cargo cinstall -p yara-x-capi --release
-```
-
-Install malcontent:
-```sh
-go install github.com/chainguard-dev/malcontent/cmd/mal@latest
-```
+`yara-x` requires an underlying C API to function with Golang.
```

Comment:
I find this text confusing - it isn't clear why it's relevant yet what yara-x requires.

Reply:
Removed in [`d39a001` (#835)](https://github.com/chainguard-dev/malcontent/pull/835/commits/d39a00102517a38d70952af23d6d0fc41c86ce3f).